homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -66,14 +69,17 @@
                         Envoy(
                             ip_address,
                             condition,
-                            "{} {}".format(SENSORS[condition][0], inverter),
+                            "{}{} {}".format(name, SENSORS[condition][0], inverter),
```

Comment:
Please use f-strings instead of str.format.

Reply:
Done